template/base/src/env/env-schema.mjs

```diff
@@ -11,12 +11,13 @@ export const clientEnvSchema = z.object({
 });
 
 /**
- * Next.js client-side environment variables are evaluated at build time, 
+ * Next.js client-side environment variables are evaluated at build time,
  * so only environment variables actually used will be included.
- * 
+ *
  * Define your client-side environment variables in this object to
- * be able to use autocompletion and destructuring in your code. 
+ * be able to use autocompletion and destructuring in your code.
+ * @type {{ [k in keyof z.infer<typeof clientEnvSchema>]: z.infer<typeof clientEnvSchema>[k] | undefined }}
```

Comment:
This is the relevant change, which is also present in the other versions of `env-schema`